Assets/Scripts/ChargingPig.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ChargingPig : EnemyController
+{
+    private bool animationPlayed;
+    private float animationDelay = 1.5f;
+    protected override void AttackPlayer()
```

Comment:
This method is called when player is close. When does the real attack happen (AKA damaging player)

Reply:
Not yet implemented in this branch, waiting for the player prefab to see what to implement.